tstime/mono/mono.go

```diff
@@ -0,0 +1,113 @@
+// Copyright (c) 2021 Tailscale Inc & AUTHORS All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package mono provides fast monotonic time.
```

Comment:
Maybe flesh out this package comment a bit with some numbers and a big scary warning that it should almost never be used and std time is plenty fast?

Reply:
Done